Tekst ABC

okpedersen opened the pull request:
Kort level 1 oppgave. Ment som en "opptakt" til mer avanserte strengoppgaver.

okpedersen committed:
Ny oppgave: Tekststrenger 1

arve0 commented on src/python2/tekststrenger_1/tekststrenger_1.md:
````diff
@@ -0,0 +1,165 @@
+
+```python
+>>> s = "Per og Ada"
+>>> print(s.upper())        # Bare store bokstaver
````
Er det bevist at du bruker `print` i REPL?

arve0 committed:
omdøp til Tekst ABC

Tekst 1, 2, 3 kan bli litt drygt i lengden. Føler ABC gir et personlig preg,
dog introduksjon, intro, osv kan også brukes.

arve0 committed:
språklige endringer, print i REPL

språk:
Subjektivt. Generelt:
- unngå (unødige) faguttrykk og vanskelige ord
- unngå påstander som "den enkleste"
- unngå "du skal", høres ut som en befaling

print:
Slik jeg ser det er poenget med å bruke REPL å lære brukeren å inspisere
variabler, finne metoder på objekt osv. `print` gjemmer litt av dette med å
ikke skrive ut `'` for strenger osv.`'`

okpedersen committed:
Mindre setningsendringer

okpedersen committed:
Merge branch 'arve0-vg_python_tekststrenger_1' into vg_python_tekststrenger_1

okpedersen committed:
La til avsnitt om .replace()

okpedersen commented:
Merget https://github.com/okpedersen/oppgaver/pull/3, gjorde noen mindre endringer på setningene i 878a182. La til et ekstra avsnitt om `.replace()` så vi oppnår [forkunnskapskravene til hangman](https://github.com/okpedersen/oppgaver/blob/vg_python_hangman/src/python2/hangman/README.md)
